src/Domain.LinnApps/PurchaseOrders/PurchaseOrderService.cs

```diff
@@ -537,7 +622,7 @@ private void AddDeliveryToDetail(PurchaseOrder order, PurchaseOrderDetail detail
                                                         QuantityOutstanding = detail.OurQty,
                                                         QtyNetReceived = 0,
                                                         QtyPassedForPayment = 0,
-                                                        RescheduleReason = string.Empty
+                                                        RescheduleReason = string.Empty // todo make this Requested? Check old form
```

Comment:
have sent andrew mcv a message asking about this and the date advised lead time bit